cloud/deploys/bin/azure/get-bastion-commands

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+
+# DOC: Print out command to ssh onto bastion machine
+# example: ../roc/bin/ssh-connection -g sgdev -d civiform-artistic-halibut
+key_name="bastion"
+while getopts g:d:k:p flag
+do
+    case "${flag}" in
+        g) resource_group=${OPTARG};;
+        d) database_name=${OPTARG};;
+        k) key_name=${OPTARG};;
+    esac
+done
+
+if [[ ! -z "${resource_group}" ]] || [[ ! -z "${database_name}" ]]; then
+  echo "arguments -g and -d must be provided"
```

Comment:
suggestion (blocking): print to stderr for error messages

Reply:
can use `out::error` from the lib code

Reply:
done